_health-care/_js/utils/validations.js

```diff
@@ -94,7 +92,10 @@ function isValidVeteranAddress(data) {
 }
 
 function isValidSpouseInformation(data) {
-  return (isBlank(data.spouseSocialSecurityNumber) || isValidSSN(data.spouseSocialSecurityNumber)) &&
+  return (isBlank(data.spouseFullName.first) || isValidName(data.spouseFullName.first)) &&
+      (isBlank(data.spouseFullName.middle) || isValidName(data.spouseFullName.middle)) &&
+      (isBlank(data.spouseFullName.last) || isValidName(data.spouseFullName.last)) &&
+      (isBlank(data.spouseSocialSecurityNumber) || isValidSSN(data.spouseSocialSecurityNumber)) &&
       ((isBlank(data.spouseDateOfBirth.day) && isBlank(data.spouseDateOfBirth.month) && isBlank(data.spouseDateOfBirth.year)) ||
       isValidDate(data.spouseDateOfBirth.day, data.spouseDateOfBirth.month, data.spouseDateOfBirth.year)) &&
       ((isBlank(data.dateOfMarriage.day) && isBlank(data.dateOfMarriage.month) && isBlank(data.dateOfMarriage.year)) ||
```

Comment:
These validations are getting hard to read. Maybe a validator of `isEnoughName()` could shrink the validator a bit? Good enough for now though.